src/nccl_ofi_scheduler.c

```diff
@@ -91,12 +80,8 @@ static inline int set_schedule_by_threshold(nccl_net_ofi_threshold_scheduler_t *
 					    nccl_net_ofi_schedule_t *schedule)
 {
 	int ret = 0;
-	if (size > scheduler->rr_threshold) {
-		nccl_net_ofi_set_multiplexing_schedule(size, num_rails,
-						       align, schedule);
-	} else {
-		ret = set_round_robin_schedule(scheduler, size, num_rails, schedule);
-	}
+	int num_stripes = (int) NCCL_OFI_MIN(NCCL_OFI_DIV_CEIL(size, scheduler->min_stripe_size), num_rails);
+	nccl_net_ofi_set_multiplexed_round_robin_schedule(scheduler, size, num_rails, num_stripes, align, schedule);
```

Comment:
There's no point to nccl_net_ofi_set_multiplexed_round_robin_schedule.  The body of that function should just be here.

Reply:
Done